examples/jenkins/master-slave/README.md

```diff
@@ -1,97 +1,37 @@
 Jenkins Master->Slave Example
```

Comment:
this is going to be the only file left in this directory, right?  maybe we should just append this content to the jenkins/README.md as a secondary tutorial/extended example?